.github/workflows/anthropic.yml

```diff
@@ -15,7 +15,7 @@ defaults:
     working-directory: integrations/anthropic
 
 concurrency:
-  group: cohere-${{ github.head_ref }}
+  group: anthropic-${{ github.head_ref }}
```

Comment:
unrelated: I noticed that Anthropic tests were not running due to the wrong concurrency group.